### PR TITLE
Onboarding updates

### PR DESCRIPTION
**Pull request by mavarius**
Closes influxdata/applications-team-issues#146
Closes influxdata/applications-team-issues#147
Closes influxdata/applications-team-issues#148
Closes influxdata/applications-team-issues#154

_Briefly describe your proposed changes:_
Removes icon and header from top of all wizard cards and changes copy in footer.
Fixes styling on screens with limited vertical space.
Adds Welcome step.
Removes skip button from final step.

  - [ ] Rebased/mergeable
  - [ ] Tests pass

### CODE REVIEW

File: ui/src/sources/components/WelcomeStep.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+}
+
+@ErrorHandling
+class SourceStep extends PureComponent<Props> {
+  constructor(props: Props) {
+    super(props)
+  }
+
+  public next = async () => {
+    // navigate to newly created source?
```
Comment (ischolten) [inline ui/src/sources/components/WelcomeStep.tsx]: should something be happening here? or is this part tbd?

File: ui/src/sources/containers/OnboardingWizard.tsx [hunk 2]
```diff
@@ -141,6 +155,11 @@ class OnboardingWizard extends PureComponent<Props, State> {
     )
   }
 
+  // WelcomeStep
+  private isWelcomeComplete = () => {
+    return true
```
Comment (ischolten) [inline ui/src/sources/containers/OnboardingWizard.tsx]: should this always return true?

File: ui/src/sources/components/WelcomeStep.tsx (hunk 1 above)
Comment (mavarius) [reply ui/src/sources/components/WelcomeStep.tsx]: I think this was tbd, it's a note from Deniz

File: ui/src/sources/containers/OnboardingWizard.tsx (hunk 2 above)
Comment (mavarius) [reply ui/src/sources/containers/OnboardingWizard.tsx]: Yes, this first step is always complete.

**Pull request merged**